src/lib/forward.ts

```diff
@@ -95,8 +112,14 @@ export class EventForwarder {
 
     node.on("ready", (changedNode: ZWaveNode) => {
       // Dump full node state on ready event
-      const nodeState = dumpNode(changedNode);
-      notifyNode(changedNode, "ready", { nodeState });
+      this.clients.clients.forEach((client) =>
+        this.sendEvent(client, {
+          source: "node",
+          event: "ready",
+          nodeId: changedNode.nodeId,
+          ...dumpNode(changedNode, client.schemaVersion),
```

Comment:
You sure about the `...` ? Shouldn't it be `node: dumpNode(…)` ?

Reply:
That's how we do it for other times we send over the node. Otherwise we can't distinguish what is event message vs what is the node.

Reply:
Yes, this is the additional data

Reply:
I think you're right. It was automagically handled by JS but could to explictly return it

Reply:
![image](https://user-images.githubusercontent.com/1444314/109206534-d473fd80-775c-11eb-976f-1fb8fc37194f.png)

All these properties are now added to the event object instead of being under `node`

Reply:
fixed!